ibis/backends/exasol/__init__.py

```diff
@@ -253,50 +253,47 @@ def _register_in_memory_table(self, op: ops.InMemoryTable) -> None:
                 f"got null typed columns: {null_columns}"
             )
 
-        # only register if we haven't already done so
-        if (name := op.name) not in self.list_tables():
```

Comment:
The only bits I changed here were to remove this conditional, and then dedent what was previously indented inside the branch.